CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ if(NOT BUILD_LIBCREATEREPO_C_SHARED)
   set(CMAKE_POSITION_INDEPENDENT_CODE 1)
 endif()
 
+option(WITH_ZSTD "Build with zstd support" ON)
```

Comment:
Add this to the build deps section of the README

`* libzstd (<http://facebook.github.io/zstd/>) - libzstd-devel/libzstd-dev`